src/domain/shared/components/DashboardSections/DashboardAddButton.tsx

```
@@ -28,13 +31,14 @@ const DashboardNavigationAddSubspace = ({ url, level = 0, onClick }: DashboardNa
           <AddCircleOutlineOutlined />
         </Avatar>
       }
-      padding
       square
-      sx={getIndentStyle(level)}
+      sx={{ ...getIndentStyle(level), ...sx }}
     >
-      <CaptionSmall>{t('buttons.addSubject', { subject: t('common.subspace') })}</CaptionSmall>
+      <CaptionSmall>
+        {Boolean(translationKey)
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)